s30900-Pawel_Kozlowski/Zadania/2.Linux - Uprawnienia dostępu.txt

```diff
@@ -65,16 +64,16 @@
    │   └── plik.txt
    └── zadania [error opening dir]
 
-3 directories, 0 files
+   3 directories, 0 files
    s30900@szuflandia22:~/katalog$ chmod u+r,u+w,u+x zadania
 
-   Czy możemy bezpośrednio przeskoczyć do katalogu ABC/XYZ, jeśli nie mamy uprawnienia wstępu do ABC, ale mamy do XYZ ?
+   Czy można bezpośrednio przeskoczyć do katalogu ABC/XYZ, jeśli nie mamy licencji wstępnej do ABC, ale mamy do XYZ?
 
-   gdy urzyjemy ścieżki bezwzględnej to możemy przejść do katalogu ponieważ omijamy katalog Do którego nie mamy dostępu
-   ścieżka względna by wypisała błąd ponieważ nie mamy dostępu przejść przez katalog ABC
+   Gdy używamy bezwzględnego ścieżki dostępu do katalogu głównego, pomijamy katalog, do którego nie mamy dostępu.
```

Comment:
To zdanie nie ma składu.
Poprzednie było lepsze choć zawierało błędy ortograficzne.